packages/devtools/test/integration_tests/util.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Note I had to move this because these are used by web tests, but the previous location imports dart:io which isn't allowed in build_web_compilers 2.0.